packages/sdk/src/version.js

```diff
@@ -0,0 +1,2 @@
+// NOTE: This string will be replaced in the a `prepare` run script by the `scripts/version.js` file.
```

Comment:
can we fix this typo before we put it all over the place?

Reply:
Which typo? I copied it from the other PR...

Reply:
I think he just based off my PR.

Reply:
```suggestion
// NOTE: This string will be replaced in the `prepare` script by the `scripts/version.js` file.
```